app/src/main/java/com/radu/newsreader/model/article/ArticleItemViewModel.java

```diff
@@ -0,0 +1,19 @@
+package com.radu.newsreader.model.article;
+
+import androidx.lifecycle.LifecycleObserver;
+import androidx.lifecycle.ViewModel;
+
+public class ArticleItemViewModel extends ViewModel implements LifecycleObserver {
```

Comment:
ArticleItemViewModel list of items is held, created and injected trough the NewsListViewModel, it's not injected on it's own, therefore we don't need to extend the ViewModel for each and every item. Also implements LifecycleObserver is useless as we're not subscribing or using any lifecycle events.